### DISCUSSION TITLE
BLAS not found when installing in Docker

### DISCUSSION THREAD

**Discussion by braaannigan** [alan-turing-institute/sktime#2762] [Q&A] [answered]
Hi - I'm trying to pip install sktime in a Debian ubuntu image (the python:3.8 image). When I run pip install sktime I get lots of errors about not being able to find BLAS e.g. "Optimized (vendor) Blas libraries are not found."

Does anyone know how to remedy this?

Liam

**Comment by braaannigan** [comment, marked as answer]
I've re-loaded and it's working now. Not sure what happened tbh

**Reply by fkiraly** [reply]
well, sometimes mysterious bugs like this happen and they disappear again...

**Comment by aiwalter** [comment]
Pls provide error message
